src/cavendish_particle_tracks/_decay_angles_dialog.py

```diff
@@ -55,30 +53,83 @@ def __init__(self, parent=None):
         ):
             self.layout().addWidget(table_datum, i % 3 + 2, i // 3)
 
-        self.layout().addWidget(bss, 4, 0, 1, 3)
-
+        self.layout().addWidget(btn_calculate, 5, 0, 1, 3)
         self.layout().addWidget(
             QLabel("Opening angles"),
-            5,
+            6,
             0,
             1,
             3,
         )
         for i, widget in enumerate(
             [QLabel("ϕ_proton [rad]"), QLabel("ϕ_pion [rad]")] + self.textboxes_phi
         ):
-            self.layout().addWidget(widget, i % 2 + 6, i // 2 + 1)
-        self.layout().addWidget(bap, 8, 0, 1, 3)
-        self.layout().addWidget(self.buttonBox, 10, 0, 1, 3)
-
+            self.layout().addWidget(widget, i % 2 + 7, i // 2 + 1)
+        self.layout().addWidget(btn_save, 9, 0, 1, 3)
```

Comment:
This looks like minor UI changes that we probably don't want conflated with this feature.

Reply:
There are some changes needed in the UI, as I needed to add a new combobox for the choice of `origin` vs `decay` vertex. Therefore, I'd go ahead and update the button names.